src/EducationService.Validation/Education/CreateEducationRequestValidator.cs

```diff
@@ -23,21 +26,23 @@ public class CreateEducationRequestValidator : AbstractValidator<CreateEducation
     {
       _rcCheckUsersExistence = rcCheckUsersExistence;
       _logger = logger;
+      
+      Thread.CurrentThread.CurrentUICulture = new CultureInfo("ru-RU");
 
       RuleFor(x => x.UserId)
         .Cascade(CascadeMode.Stop)
-        .NotEmpty().WithMessage("Wrong user id value.")
+        .NotEmpty().WithMessage($"{nameof(CreateEducationRequest.UserId)} {EducationValidatorResource.IsEmpty}")
         .MustAsync(async (pu, cancellation) => await CheckValidityUserId(pu, new List<string>()))
-        .WithMessage("User does not exist.");
+        .WithMessage($"{nameof(CreateEducationRequest.UserId)} {EducationValidatorResource.UserDoesNotExist}");
```

Comment:
may be without {nameof(CreateEducationRequest.UserId)}?
what would be the full message?